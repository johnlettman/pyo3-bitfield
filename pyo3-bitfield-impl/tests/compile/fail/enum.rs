use pyo3_bitfield_impl::PyBitfield;

/// Attempt the [pyo3_bitfield_impl::PyBitfield] derive on an unsupported enum type.
#[derive(PyBitfield)]
enum TestEnum {
    Item1,
    Item2,
    Item3,
    Item4
}

fn main() {}