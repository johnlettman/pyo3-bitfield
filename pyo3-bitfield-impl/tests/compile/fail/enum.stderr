error: item is not a struct

         = help: `#[derive(PyBitfield)]` only supports structs

 --> tests/compile/fail/enum.rs:5:6
  |
5 | enum TestEnum {
  |      ^^^^^^^^
